woocommerce/woocommerce-ios
# Coupons: Add coupon creation success screen

Part of #6493 


### Description

This PR adds a new screen to be displayed when coupon creation succeeds. This includes UI for the view, action handling for the bottom buttons, and animations for the view (following the design concept in p91TBi-7FL-p2). The view will need to be integrated into the Coupon Creation view later with input for the coupon code and closure for when the view is dismissed. Analytics will also be added in the next PR for easier testing.

### Testing instructions

Since the view hasn't been integrated, please test this PR by building it in Xcode and testing the preview. You should be able to see that the share button presents the share sheet, and view components are animated upon appearing.

### Screenshots





https://user-images.githubusercontent.com/5533851/172592517-736c5cc9-669d-4592-8121-5e4dea144d41.mov









---
